assets/css/style.css

```diff
@@ -98,3 +99,178 @@ body {
   display: flex;
   justify-content: center;
 }
+
+.cards-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 320px);
+  gap: 26px;
+  place-content: center;
+}
+
+.product-card {
+  width: 320px;
+  border: solid 1px var(--very-light-black);
+  border-radius: 10px;
+}
+
+.product-card img {
+  width: 100%;
+  height: 200px;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+  object-fit: cover;
+}
+
+.product-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  height: 200px;
+}
+
+.product-info p {
+  margin: 0;
+  font-size: 25px;
+  font-weight: 600;
+  text-align: center;
+}
+
+.product-info button {
+  background-color: var(--green);
+  color: var(--white);
+  border-color: var(--green);
+  border-radius: 5px;
+  width: 122px;
+  height: 50px;
+  font-weight: 500;
+  font-size: var(--md);
```

Comment:
- [ ] Kindly make sure the **Get My Resume** `button` follow the button styling: `Active`, `Hover`, and `Pressed` (see the design). Take a look at the previous pic where I pointed out the other button and check the correct colors from the Figma design.